l10n_ch_hr_payroll/README.rst

```diff
@@ -9,9 +9,10 @@ This module allows you to manage the salaries of your employees
 ** Features list :**
     * Add Swiss salary rule categories
     * Add Swiss salary rules
+    * Make code, name and note translatable in salary rules and salary rules categories
     * Add children in school to employee
-    * Add LPP range to contract
-    * Add LPP Amount to contract.
+    * Add OBP (Occupational Benefit Plan/LPP or BVG) range to contract
+    * Add OBP Amount to contract.
```

Comment:
Are OBP & LPP the same thing?
If it is so, why changing it on all documents / classes ?